src/Data/Impact/Definition.elm

```diff
@@ -69,36 +70,137 @@ type alias Definition =
     }
 
 
-type alias Definitions =
-    { acd : Definition
-    , bvi : Definition
-    , cch : Definition
-    , ecs : Definition
-    , etf : Definition
-    , etfc : Definition
-    , fru : Definition
-    , fwe : Definition
-    , htc : Definition
-    , htcc : Definition
-    , htn : Definition
-    , htnc : Definition
-    , ior : Definition
-    , ldu : Definition
-    , mru : Definition
-    , ozd : Definition
-    , pco : Definition
-    , pef : Definition
-    , pma : Definition
-    , swe : Definition
-    , tre : Definition
-    , wtu : Definition
+type alias DefinitionsBase a =
```

Comment:
It's probably worth adding a comment explaining that we need accepting either definitions or impacts.